app/controllers/candidate_interface/course_choices/replace_choices/decision_controller.rb

```diff
@@ -9,6 +9,9 @@ def choose_action
           @course_name_and_code = @course_choice.course.name_and_code
           @provider_name = @course_choice.provider.name
           @site_name = @course_choice.site.name
+          @study_mode = @course_choice.course_option.study_mode.humanize.downcase
+          @other_locations_available = create_pick_site_form(@course_choice, @course_choice.course_option).available_sites.present?
+          @other_study_mode = @course_choice.course_option.get_alternative_study_mode
```

Comment:
This looks good and works for me locally (when I tried manually changing the availability in my database) but there is one (edge) case that isn't covered. When the course I applied to no longer has vacancies but another location has vacancies with a different study mode I am just told that the course is full. I'm not sure this covered in the design but it doesn't seem right to say that the whole course is just full.